.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules\\typescript\\lib"
```

Comment:
Why is it needed?

Reply:
sets that when opened in vscode uses the typescript version specified as a dependency as opposed to the typescript version vscode ships with ( we houls have this in every ts project we have )

Reply:
To make sure that VSCode is using the TypeScript version.

Reply:
I see. I didn't know that. I'll add it to our projects.